.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
     "debug.internalConsoleOptions": "neverOpen",
     "azureFunctions.preDeployTask": "publish (functions)",
     "omnisharp.enableEditorConfigSupport": true,
-    "omnisharp.enableRoslynAnalyzers": true
+    "omnisharp.enableRoslynAnalyzers": true,
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
why are java changes showing up?

Reply:
I merged main, not sure why these are showing up, checking

Reply:
Remove this change too please